README.md

```diff
@@ -0,0 +1,93 @@
+
+## Overview
+
+This walkthrough will guide you through steps to configure SPIRE Server and SPIRE Agent for AWS IID based attestation.
```

Comment:
Since this is the README for the whole repo, might be worth quickly explaining what the AWS IID attestor plugin does. eg.

_The AWS IID attestor is a plugin for the SPIRE Agent and SPIRE Server that allows SPIRE to automatically attest instances using the AWS Instance Metadata API and the AWS Instance Identity document. It also allows an operator to use AWS Instance IDs when defining SPIFFE ID attestation policies. This plugin is also a pre-requisite for the AWS node resolver plugin._